nexus/types/src/deployment/blueprint_diff.rs

```diff
@@ -731,11 +733,14 @@ impl ModifiedZone {
         if reason.is_empty() {
             Ok(ModifiedZone {
                 prior_disposition: *diff.disposition.before,
+                prior_image_source: diff.image_source.before.clone(),
                 zone: BlueprintZoneConfig {
                     disposition: *diff.disposition.after,
                     id: *diff.id.after,
                     filesystem_pool: diff.filesystem_pool.after.cloned(),
                     zone_type: diff.zone_type.after.clone(),
+                    // XXX: Is it an error if the image source is modified?
```

Comment:
Maybe remove this comment now given our current disposition toward this?

Reply:
Done, thanks.